optimum/tpu/cli.py

```diff
@@ -0,0 +1,113 @@
+import importlib.util
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+
+import click
+import typer
+
+
+TORCH_VER = "2.4.0"
```

Comment:
Should you source the version from `torch` directly? Seems very error prone / easy forgettable in the long run isn't it?

Reply:
I was actually thinking in the long run I will try to remove the dependency altogether, so users could install `optimum-tpu` and then install torch cpu using this CLI: this will allow to avoid installing the cuda dependencies of the default torch package.